api/core/tools/entities/tool_entities.py

```diff
@@ -147,7 +147,7 @@ class ToolParameterType(StrEnum):
         FILES = "files"
 
         # deprecated, should not use.
-        SYSTEM_FILES = "systme-files"
+        SYSTEM_FILES = "system-files"
```

Comment:
this we found before but the better way is do not changed that, some users use this typo dir...

Reply:
OK.